modules/registry-common-terms.adoc

```diff
@@ -24,8 +24,8 @@ namespace::
 A namespace isolates groups of resources within a single cluster.
 
 ifndef::openshift-rosa[]
```

Comment:
Prior to this change, I had to add a duplicate entry for ROSA. That's because this entry used to be "`{product-title}` registry", which resolved to a different name for ROSA only (which consequently meant that I had to add the duplicate ROSA-only entry to maintain the alphabetic order of the glossary).

Now that the name of the registry is the same for all distributions, you should be able to remove the conditionalization here and also delete the ROSA-specific entry starting on line 43 (https://github.com/openshift/openshift-docs/pull/58485/files#diff-8ee1d28fc512988e3a3019de2ef3c11b825330e0c6ee9f577c508dd04f88b083R43).

Reply:
Fixed